helm/ph-ee-test/values.yaml

```diff
@@ -422,7 +446,71 @@ ph-ee-engine:
       host: "operationsmysql"
       port: 3306
       schema: "tenants"
+  
+  integration_test:
+    enabled: true
+    replicas: 1
+    image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-integration-test"
+    imageTag: latest
+    imagePullPolicy: "Always"
+    limits:
+      cpu: "500m"
+      memory: "3Gi"
+    requests:
+      cpu: "100m"
+      memory: "2Gi"
+    operations_app:
+      contactpoint: "https://ops-bk2.sandbox.fynarfin.io"
+    bulk_processor:
+      contactpoint: "https://bulk-connector2.sandbox.fynarfin.io"
+    channel_connector:
+      contactpoint: "https://channel2.sandbox.fynarfin.io"
+    kafka:
+      brokers: "kafka:9092"
+      topics: "zeebe-export"
+      consumerTimeoutMs: "300000"
+    zeebe_operations:
+      contactpoint: "https://zeebeops2.sandbox.fynarfin.io"
+      no_of_workflows: "100"
+      thread-count: "20"
+    max_retry_count: "5"
+    retry_intervals: "15000"
+    defaults:
+      tenant: "gorilla"
+      authorization: "Basic bWlmb3M6cGFzc3dvcmQ="
+    channel:
+      base_url: "https://channel2.sandbox.fynarfin.io"
+    loan:
+      base_url: "https://fynams2.sandbox.fynarfin.io"
+    savings:
+      base_url: "https://fynams2.sandbox.fynarfin.io"
+    gradle:
+      command: "./gradlew cucumberCli -P tags=@ZeebeExport;"
 
   wildcardhostname: "*.sandbox.fynarfin.io"
 
   tls: ""
+
+  keycloak:
+    enabled: false
+
+  kong:
+    enabled: false
+
+  redis:
+    enabled: false
+  
+zeebe-operate:
+  ingress:
+    enabled: true
+    hostname: "zeebeoperate.sandbox.fynarfin.io"
```

Comment:
A lot of these contactpoints and hostnames are not matching with integration test hostnames / contact points.
Ex:- Integration test config https://ops-bk2.sandbox.fynarfin.io/ is not matching with contact point & hostname https://ops-bk.sandbox.fynarfin.io/. Keep https://ops-bk2.sandbox.fynarfin.io/ consistent.